### ISSUE TITLE
[Feature Request] Landscape Mode

### ISSUE THREAD

**Issue by putulopi** [ProtonMail/proton-mail-android#76]

**Comment by Ammako** [comment]
Landscape mode works fine here, what is the issue?

**Comment by putulopi** [comment]
The Pin Lock doesn't follow the rotation of the screen, its only available on portrait :)

**Comment by julian-fairfax** [comment]
I'd also like to add to this that the app's launch screen itself (pin or no pin) rotates the screen to portrait even though my tablet is in landscape mode and the app supports landscape mode. Thanks for your help and thanks for developing this wonderful app!

**Comment by Mr-Bajs** [comment]
Yes, the pin-lock works but if you have the device in landscape mode, then the Pin-Lock is only in portrait mode. Seams like the app is having the pin-lock locked in portrait mode regardless of the device configuration.
